benches/barchart.rs

```diff
@@ -16,12 +16,12 @@ pub fn barchart(c: &mut Criterion) {
         let data: Vec<Bar> = (0..data_count)
             .map(|i| {
                 Bar::default()
-                    .label(format!("B{i}").into())
+                    .label(format!("B{i}"))
                     .value(rng.gen_range(0..data_count))
```

Comment:
All the examples should use the `::new` function instead of default in my opinion.